### PR TITLE
app.js clean-up ++ CSS Changes

### PR DESCRIPTION
**Pull request by CharlieWillingale**
Cleaned up map function in app.js so each is its own li, whereas before each was creating a new ul with an li within.
Added simple styling to create 'cards' for each route (similar to bootstrap) - incomplete.
Removed all horrible default CSS styling n(margin, padding, list-style etc.)

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import styles from '../styles/global.css'
+import '../styles/global.css'
```
Comment (boda1) [inline pages/_app.js:1]: What's the reason for this change? Tidier formatting?

File: pages/index.js [hunk 2]
```diff
@@ -5,6 +5,8 @@ import Filters from '../components/filters.js'
 // import Routes from '../components/routes.js'
 import { promises as fs } from 'fs';
 import path from 'path';
+import styles from 'mapbox/lib/services/styles';
```
Comment (boda1) [inline pages/index.js:8]: Unused but added in preparation for embedding mapbox maps?

File: pages/index.js [hunk 3]
```diff
@@ -15,11 +17,13 @@ export default function Home({ routesList }) {
...
-          <li>{route.route}</li>
-        </ul> 
-          )}
+      <ul className="route-cards" >
+        {routesList.routes.map(route =>  
+          <li className='route-card' key={route.id}>
+            {route.route}
+          </li>
+        )}
+      </ul>
```
Comment (boda1) [inline pages/index.js:26]: This was very silly from me!

File: pages/_app.js (hunk 1 above)
Comment (CharlieWillingale) [reply pages/_app.js:1]: I don't think the former imported the style sheet correctly.

File: pages/index.js (hunk 2 above)
Comment (CharlieWillingale) [reply pages/index.js:8]: I don't think I added this, must have been either when I installed or some auto text
